cmd/kubeseal/main.go

```diff
@@ -618,6 +618,9 @@ func run(w io.Writer, secretName, controllerNs, controllerName, certURL string,
 	if unseal {
 		return unsealSealedSecret(w, os.Stdin, scheme.Codecs, privKeys)
 	}
+	if len(privKeys) != 0 && isatty.IsTerminal(os.Stderr.Fd()) {
+		fmt.Fprintf(os.Stderr, "warning: you passed --recovery-private-key without also choosing the unseal command with --recovery-unseal, are you sure?\n")
```

Comment:
the "are you sure?" bit is a bit weird.

Reply:
indeed; I hoped you could help me come up with a better phrasing :-)